pkg/liquidity-mining/test/ChildChainGauge.test.ts

```diff
@@ -137,6 +137,10 @@ describe('ChildChainGauge', () => {
         const tx2 = await gauge.connect(user2).user_checkpoint(user2.address);
         expectEvent.notEmitted(await tx2.wait(), 'Transfer');
       });
+
+      it('can be performed by any address', async () => {
+        await expect(gauge.connect(other).user_checkpoint(user1.address)).to.not.be.reverted;
```

Comment:
You can check that the `UpdateLiquidityLimit` event is emitted (which is the reason why we care about this)